scripts/run_generator.sh

```diff
@@ -0,0 +1,17 @@
+##!/bin/bash
+
+export OutputDir=/global/cfs/cdirs/m4287/hep/Delphes_PYTHIA8_output
+export WorkDir=/global/cfs/cdirs/m4287/hep/genHEPdata
+
+shifter --image=rootproject/root:6.28.04-ubuntu22.04 /bin/bash $WorkDir/scripts/generate_events.sh $1 $2
```

Comment:
you shouldn't need the `--image` option here if it is set in the job's `#SBATCH` declarations. Anyway, the image version should probably match.